main.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+installation_script="$(mktemp)"
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/48339106eb0d403a3c66519317488c8185844b32/install-poetry.py --output "$installation_script"
+
+if [ "${RUNNER_OS}" == "Windows" ]; then
+    path="C:/Users/runneradmin/AppData/Roaming/Python/Scripts"
+else
+    path="$HOME/.local/"
+fi
+
+echo -e "\n\033[33mSetting Poetry installation path as $path\033[0m\n"
+echo -e "\033[33mInstalling Poetry 👷\033[0m\n"
+
+if [ "${VERSION}" == "latest" ]; then
+    POETRY_HOME=$path python3 $installation_script --yes ${INSTALLATION_ARGUMENTS}
+else
+    POETRY_HOME=$path python3 $installation_script --yes --version=${VERSION} ${INSTALLATION_ARGUMENTS}
+fi
```

Comment:
Assuming the condition becomes redundant if we use `POETRY_` env vars
```suggestion
POETRY_HOME=$path python3 $installation_script --yes ${INSTALLATION_ARGUMENTS}
```

Reply:
On second thought, I'm not so sure the version will be inferred by the installer. I can help test tomorrow.

Reply:
`POETRY_VERSION=1.1.10 python3 $installation_script --yes` prompts for a 1.1.10 installation, so that seems like it's working 👍